docs/Maths.md

```diff
@@ -226,6 +226,107 @@ Result:
 ![](plotly_svg.png)
 
 
+## Using Pygal ##
+[Pygal](https://www.pygal.org/en/stable/) is a graph plotting library using python. You can install Pygal using ` $ pip install pygal` command. 
+
+`fpdf2` is able to embed the graph and charts that are generated using `Pygal` library. The following ways explain how to embed `Pygal` charts into `fpdf2` library.
+
+### Using cairosvg (*A faster and efficient implementation*) ### 
+
+A faster and expected approach of embedding a `Pygal` svg graph into a PDF file is to use the `cairosvg` library to convert the `svg` string generated by `pygal` into byte string using `BytesIO` library so that we can keep these data in an in-memory buffer. 
+As the `fpdf` library can understand byte string of a `svg` content, it can easily embed a graph inside a `pdf`. 
+
+```python
+import pygal
+from fpdf import FPDF
+from io import BytesIO
+import cairosvg
+
+# Create a Pygal bar chart
+bar_chart = pygal.Bar()
+bar_chart.title = 'Browser usage evolution (in %)'
+bar_chart.x_labels = map(str, range(2002, 2013))
+bar_chart.add('Firefox', [None, None, 0, 16.6,   25,   31, 36.4, 45.5, 46.3, 42.8, 37.1])
+bar_chart.add('Chrome',  [None, None, None, None, None, None,    0,  3.9, 10.8, 23.8, 35.3])
+bar_chart.add('IE',      [85.8, 84.6, 84.7, 74.5,   66, 58.6, 54.7, 44.8, 36.2, 26.6, 20.1])
+bar_chart.add('Others',  [14.2, 15.4, 15.3,  8.9,    9, 10.4,  8.9,  5.8,  6.7,  6.8,  7.5])
+
+# Use CairoSVG to convert PNG from SVG of barchart
+svg_img_byte = BytesIO()
```

Comment:
```suggestion
svg_img_bytesio = BytesIO()
```
This is just a suggestion: do you think this naming would be clearer?

Reply:
Right. Thanks for the review. I will edit and commit this change soon.